src/Components/Home.js

```diff
@@ -19,20 +24,49 @@ function Home() {
       .finally(() => setShowProgress(false))
   }, [])
 
+  //  Get Avatar
+  useEffect(() => {
+    list.forEach((user) => {
+      const splitId = user.avatar.split('/')
+      const userID = splitId[3].split('.')
+      const githubID = userID[0]
+      const url = 'https://avatars.githubusercontent.com/' + githubID
+      fetch(url)
+        .then(response => response.blob())
```

Comment:
I think could be achieved with events, here is an example  https://github.com/EddieHubCommunity/LinkFree/pull/447/files#diff-4ed8433b2115f0e747931130c6e484267e32daea768f03e3d66f76142b2cfbe0R18